openmdao/core/driver.py

```diff
@@ -10,6 +10,7 @@
 
```

Comment:
I'm kind of impressed that this didn't require any changes to Driver.

Reply:
Yeah it turns out that Driver doesn't manage the iteration of the model subsystems at all. It just tells the model to do a nonlinear solve and lets the nonlinear solver do the iteration.